vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -210,10 +234,10 @@ An `env.json` file to configure the application URLs and client credentials must
 }
 ```
 
-Instructions to export a bundle, including how to configure `env.json`, can be found in [this tutorial](../../tutorials/create/pb/export-bundle-from-application.md).
+Instructions to export a bundle, including how to configure `env.json`, can be found in the [Export and Publish a Bundle tutorial](../../tutorials/create/pb/export-bundle-from-application.md).
 
 ## Profile Management
-To manage and switch between different `ent` configurations, use `ent profile`. 
+To manage and switch between different ent configurations, use `ent profile`. 
```

Comment:
ent here should be backticked. It's not really a thing on its own, it's part of a configuration here and commands elsewhere.

Reply:
i disagree; ent is it's own thing. it's the entando command line interface. there are many references to "Entando CLI (ent)" and it's an entity (no pun intended), not a command